### ISSUE TITLE
Uncontrollable Worry

### ISSUE THREAD

**Issue by SmartMonkey-git** [obophenotype/human-phenotype-ontology#11600]
**Preferred term label:**
Uncontrollable Worry

**Synonyms**

**Definition (free text, please give PubMed ID)**
A cognitive abnormality characterized by the subjective experience that one's worry is difficult to control or dismiss.
[PMID: 23713499](https://pubmed.ncbi.nlm.nih.gov/23713499/)

**Parent term (use hpo.jax.org/app)**
Abnormal preoccupation 
HP:0025785

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
generalized anxiety disorder
MONDO:0001942

**Your nano-attribution (ORCID)**
https://orcid.org/0009-0002-1118-4784
